script/VerifyL1OFTAdapter.s.sol

```diff
@@ -30,23 +30,36 @@ contract VerifyL1OFTAdapter is BaseScript, BatchScript {
     SendLibConfig[] public newSendLibs;
     ReceiveLibConfig[] public newReceiveLibs;
 
-    function run(string calldata _jsonPath) public isBatch(getData(block.chainid).OFT_OWNER) {
-        _loadInput(_jsonPath);
+    function run(
+        string calldata _jsonPath,
+        string calldata _deploymentPath
+    )
+        public
+        isBatch(getData(block.chainid).OFT_OWNER)
+    {
+        string memory _fullDeploymentPath = string(abi.encodePacked(vm.projectRoot(), _deploymentPath));
+        _loadInput(_jsonPath, _fullDeploymentPath);
 
         require(currentDeployment.isL1 == true, "Must be L1 deployment");
 
-        console.log("Predicted L1OFTAdapter:", predictions.l1OFTAdapter);
```

Comment:
why is this deleted?

Reply:
Predictions woudn't work for older deployments therefore I had refactored the scripts to not use predicted addresses but the deployed addresses.